src/array/kernel_decl.h

```diff
@@ -96,6 +96,13 @@ void SDDMMCoo(const std::string& op,
               int lhs_target,
               int rhs_target);
 
+/*!
+ * \brief Find the src/dst/etype id based on the target.
+ */
+int get_hs_id(HeteroGraphPtr graph,
```

Comment:
No need to include it in `kernel_decl.h`, right?  This should be only used in `kernel.cc`?

Reply:
Otherwise get a compile error.

Reply:
True! Also move the docstring in kernel.cc?

Reply:
Sure.